City-of-Helsinki/drupal-helfi-strategia
# Helfi ahjo

Features:
- Organization chart page
- Organizations tree
- Taxonomy tree based on Ahjo API response
- Administration page to set the configurations and set the cronjob /admin/config/ahjo
- Option to force to resync the organizations
- A block to add the organizations tree

To test the module:
- You need to set the credentials to: /admin/config/ahjo
- Press the 'Import Data and Sync' button (you can check the data to admin/structure/taxonomy/manage/sote_section/overview)
- Go to Structure - Block layout - Place a block - Sote Section)
- To check the organization diagram you have to go to /organizationchart
- If you want to force data to resync you have to go to admin/config/ahjo/cron and press 'Run Cron Job Now' button
